README.md

```diff
@@ -5,13 +5,14 @@ WSO2 Balana Implementation
 
 |  Branch | Build Status |
 | :------------ |:-------------
-| master      | [![Build Status](https://wso2.org/jenkins/job/wso2-balana/badge/icon)](https://wso2.org/jenkins/job/wso2-balana) |
-
+| master      | [![Build Status](https://wso2.org/jenkins/buildStatus/icon?job=forked-dependencies/wso2-balana)](https://wso2.org/jenkins/job/forked-dependencies/wso2-balana) |
```

Comment:
Both links don't work anymore, has your Jenkins URL changed? Should the Travis image https://travis-ci.org/wso2/balana.svg?branch=master be used instead?